src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/JSFunctionBinding.cs

```diff
@@ -333,7 +333,7 @@ internal static unsafe void InvokeJSImportCurrent(JSFunctionBinding signature, S
 #if FEATURE_WASM_MANAGED_THREADS
                 Interop.Runtime.InvokeJSImportSync((nint)args, (nint)signature.Header);
 #else
-                Interop.Runtime.InvokeJSImport(signature.ImportHandle, (nint)args);
+                Interop.Runtime.InvokeJSImport(signature.ImportHandle, (void*)args);
```

Comment:
```suggestion
                Interop.Runtime.InvokeJSImport(signature.ImportHandle, (nint)args);
```
Seems to have been missed (will require corresponding changes in `Interop.Runtime.NativeAOT.cs`).